src/prefect/tasks/database/sqlite.py

```diff
@@ -16,35 +16,42 @@ class SQLiteQuery(Task):
         - query (str, optional): the optional _default_ query to execute at runtime;
             can also be provided as a keyword to `run`, which takes precedence over this default.
             Note that a query should consist of a _single SQL statement_.
+        - data (tuple, optional): values to use when `query` is a parametrized string. See
+            https://docs.python.org/3/library/sqlite3.html for more details.
         - **kwargs (optional): additional keyword arguments to pass to the
             standard Task initalization
     """
 
-    def __init__(self, db: str = None, query: str = None, **kwargs: Any):
+    def __init__(
+        self, db: str = None, query: str = None, data: tuple = (), **kwargs: Any
+    ):
         self.db = db
         self.query = query
+        self.data = data
         super().__init__(**kwargs)
 
     @defaults_from_attrs("db", "query")
-    def run(self, db: str = None, query: str = None):
+    def run(self, db: str = None, query: str = None, data: tuple = ()):
```

Comment:
@dsaxton did you mean to add `data` to `defautls_from_attrs` ? That might break with the `()` vs `None`

Reply:
Thanks, I missed that